Formula/mongo-c-driver.rb

```diff
@@ -30,7 +30,7 @@ class MongoCDriver < Formula
 
   def install
     cmake_args = std_cmake_args
-    cmake_args << "-DBUILD_VERSION=1.18.0-pre" if build.head?
+    cmake_args << "-DBUILD_VERSION=1.25.0-pre" if build.head?
```

Comment:
Not convinced this is needed, since it seems like `cmake` should be able to calculate the version from the Git checkout from `--HEAD` installs.